src/client/dfs/duns.c

```diff
@@ -409,6 +409,9 @@ duns_resolve_path(const char *path, struct duns_attr_t *attr)
 	ssize_t		s;
 	char		str[DUNS_MAX_XATTR_LEN];
 	struct statfs	fs;
+#ifdef LUSTRE_INCLUDE
```

Comment:
I don't get why the code below is conditional to find the Lustre headers. Does it mean that we are going to have a different behavior against a non-lustre UNS depending on whether the headers are installed or not?

Reply:
yes, you are right! I wanted to get the demo running so it is a quick and dirty one :-( and I have pushed it as is. Will rework it now. Interesting is that it did not trigger any CI error !